docs/developer-guide/coal-setup.md

```diff
@@ -406,27 +406,37 @@ services have new images.
     Unmounting USB key
     Updating cn_tools on all compute nodes
     Cleaning up gz-tools tarball
-    Updated gz-tools successfully (elapsed 13s).
+    Updated gz-tools successfully (elapsed 14s).
     ```
 
 1. Update domain name services:
 
     ```bash
     [root@headnode (coal-1) ~]# sdcadm experimental update-other
-    Updating maintain_resolvers for all vm services
-    Updating DNS domain service metadata for papi, mahi
-    Updating DNS domain SDC application metadata for papi, mahi
-    No need to update region_name for this data center
-    sapi_domain already present on node.config
-    Done.
+    Update "sdc" SAPI app metadata_schema
     ```
```

Comment:
This command only runs the steps that are necessary on a given SDC... depending on its state and age. Might be useful to keep those old example output.


Overall this is just update examples to slightly new versions of the output. There isn't a lot of value in having the commit noise to track these unless the output of a command has changed quite a lot. One such example here is 'sdcadm experimental update-agents ...' below.

Could you update this PR to drop all the changes exist for that one?